erpnext/stock/report/incorrect_balance_qty_after_transaction/incorrect_balance_qty_after_transaction.py

```diff
@@ -73,7 +73,7 @@ def get_stock_ledger_entries(report_filters):
 		"Stock Ledger Entry",
 		fields=fields,
 		filters=filters,
-		order_by="timestamp(posting_date, posting_time) asc, creation asc",
+		order_by="posting_date asc, posting_time asc, creation asc",
```

Comment:
@cpdeethree I think this is fine 

however in the past, people have suggested relying on the timestamp function instead of sorting columns individually, do you know of any real differences between these two?